### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by raghav6202**
Please review the alert component

### CODE REVIEW

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    display: flex;
+    justify-content: space-between;
+}
+
+.close-icon
+{
+    font-size: 1.25rem;
+}
+
+.fi-xwluxl-times-wide:hover
```
Comment (TejasShekar) [inline Components/Alert/alert.css:77]: What is this 😅

File: Components/Alert/styles.css [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+.left-side-nav
+{
+    display: flex;
+
+    justify-content: space-between;
+    align-items: center;
+}
+
+.right-side-nav
```
Comment (TejasShekar) [inline Components/Alert/styles.css:48]: You can add this class name in line 40 using comma as both have same props. No need to explicitly define the class.

File: Components/Alert/styles.css [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+}
+
+.component-heading
+{
+    font-size: 3rem;
+    color: var(--secondary-color);
+    font-weight: 400;
+}
+
+.component-subHeading
```
Comment (TejasShekar) [inline Components/Alert/styles.css:181]: Again, like I said in my previous review, try not to use camel case and follow one naming convention

**Review by TejasShekar — COMMENTED** [review]
UI looks great. Code looks a bit messy. Format your code so that it looks uniform.
Use comments to create sections of which part of CSS is dealing with what specific component.

File: Components/Alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline Components/Alert/alert.html:6]: not mandatory but @smile_gupta suggested this in review session
#1 you can add meta tag here ex. <meta name"author" content="your name"/>

File: Components/Alert/alert.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+      <div class="alert secondary close">This is a secondary alert
+         <i class="fi-xwluxl-times-wide close-icon"></i></div>
+      
+      <div class="alert success close"> <div class="alert-text alert">
+        <i class="fi-cwsuxl-check"></i>This is a success alert </div>
+        <div class="close-icon"><i class="fi-xwluxl-times-wide"></i></div> </div>
+
+      <div class="alert fail close"><div class="alert-text alert">
+        <i class="fi-cnsuxl-exclamation-mark-rounded"></i>This is a fail alert</div>
```
Comment (yashpatel1797) [inline Components/Alert/alert.html:71]: what does the meaning of 'cnsuxl'?

**Review by yashpatel1797 — COMMENTED** [review]
I hope the class name which you have given has a meaning that I am not aware of otherwise code is good.

File: Components/Alert/alert.css (hunk 1 above)
Comment (raghav6202) [reply Components/Alert/alert.css:77]: That's the icon's class

File: Components/Alert/styles.css (hunk 3 above)
Comment (raghav6202) [reply Components/Alert/styles.css:181]: Will update thank you Tejas

File: Components/Alert/alert.html (hunk 5 above)
Comment (raghav6202) [reply Components/Alert/alert.html:71]: That's the icon's class

File: Components/Alert/alert.html (hunk 4 above)
Comment (raghav6202) [reply Components/Alert/alert.html:6]: Okay, will do that. Thank you for the feedback

**Pull request merged**